src/apis/registerTool.js

```diff
@@ -9,6 +9,7 @@
  * @param {string} [properties.buttonGroup] Group of the tool button belongs to.
  * @param {string} [properties.tooltip] Tooltip of the tool button.
  * @param {function} [annotationConstructor] The constructor function for the annotation that will be created by the registered tool.
```

Comment:
Can we change the description to this argument to say it's deprecated, and the third argument should be used? Thanks!